src/components/Button/Button.js

```diff
@@ -38,7 +46,9 @@ Button.propTypes = {
     /** Should button be disabled or not */
     disabled: PropTypes.bool,
     /** Type of the button */
-    type: PropTypes.oneOf(['submit', 'button'])
+    type: PropTypes.oneOf(['submit', 'button']),
+    /** Providing an href will render an <a> element, styled as a button. */
+    href: PropTypes.string
```

Comment:
Alphabetical order would be nice

Reply:
did we actually agreed on that?

Reply:
At least it was mentioned more than a few times... We can't be adding all of those details in a 'guideline'. It's a matter of detail but it would be nice to keep in mind.